src/cdk-lib/bedrock/models.ts

```diff
@@ -55,23 +60,25 @@ export class BedrockFoundationModel {
 
   public static readonly TITAN_EMBED_TEXT_V1 = new BedrockFoundationModel(
```

Comment:
we should add embedding model 'amazon-titan-embed-image-v1' as well with 1024 dimension.

Reply:
amazon-titan-embed-image-v1 is not available for [knowledge base](https://aws.amazon.com/bedrock/knowledge-bases/)